CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing
+
+## Proposal for vanilla installation
+
+Most dependencies even for the extras can just be automatically installed. However, there's two exceptions: ray (as we install from URL which PyPI does not allow) and tensorflow (as we need two different versions).
```

Comment:
I'd change this to add that this is only for reproducibility of the paper results. We should add a 3rd set of extras which is called just `extras` and doesn't need these extra steps.